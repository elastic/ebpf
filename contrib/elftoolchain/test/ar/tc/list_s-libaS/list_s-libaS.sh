# $Id$
inittest list_s-libaS tc/list_s-libaS
extshar ${TESTDIR}
extshar ${RLTDIR}
runcmd "${AR} ts libaS.a" work true
runcmd "plugin/teraser -ce -t list_s-libaS libaS.a" work false
runcmd "plugin/teraser -e libaS.a" result false
rundiff true
