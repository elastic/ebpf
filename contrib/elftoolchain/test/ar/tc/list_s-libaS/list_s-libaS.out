a1.o
a2.o
a3.o
a4.o
